pkg/overlay/cache.go

```diff
@@ -114,16 +107,35 @@ type UpdateRequest struct {
 	IsUp         bool
 }
 
+// NodeDossier is the complete info that the satellite tracks for a storage node
+type NodeDossier struct {
+	pb.Node
```

Comment:
why doesn't this have a field name?

Reply:
so if you have `var node NodeDossier` you can just do `node.Id` and `node.Address`.